_posts/2016-06-06-introducing-pecas.markdown

```diff
@@ -32,8 +32,8 @@ good way to detect deviations.
 
 To build this small application we decided to use
 [Rails 4.2](http://rubyonrails.org/) and the
```

Comment:
@FionaDL Is this still accurate? We should probably create an issue (or set of issues) to upgrade this to Rails 6.0

Reply:
looks like we are still using Rails 4.2, I will add an issue.

Reply:
Looks like there is already an issue open for this: https://github.com/fastruby/pecas/issues/55